cmd/nerdctl/completion.go

```diff
@@ -160,3 +160,12 @@ func shellCompletePlatforms(cmd *cobra.Command, args []string, toComplete string
 	}
 	return candidates, cobra.ShellCompDirectiveNoFileComp
 }
+
+func shellCompleteBuildNetworks(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
This can be just in `builder_build.go`

Reply:
Thanks, I made the suggested change and added a test.